microservices/s3_to_redshift/asset_data_to_redshift.py

```diff
@@ -56,15 +57,22 @@
 handler.setFormatter(formatter)
 logger.addHandler(handler)
 
+
+def clean_exit(code, message):
+    """Exits with a logger message and code"""
+    logger.info('Exiting with code %s : %s', str(code), message)
```

Comment:
@BJenning  as @danpollock  mentioned, he would like these posted to debug. Please update.